src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1490,6 +1490,37 @@ internal static bool TryGetSpecialTypeMember<TSymbol>(CSharpCompilation compilat
             return true;
         }
 
+        /// <summary>
```

Comment:
Can we `#nullable enable` this block of new code, particularly since it deals in `null` returns.